pytorch_transformers/modeling_utils.py

```diff
@@ -78,7 +78,7 @@ def save_pretrained(self, save_directory):
         self.to_json_file(output_config_file)
 
     @classmethod
-    def from_pretrained(cls, pretrained_model_name_or_path, *input, **kwargs):
+    def from_pretrained(cls, pretrained_model_name_or_path, *args, **kwargs):
         r""" Instantiate a PretrainedConfig from a pre-trained model configuration.
 
         Params:
```

Comment:
We want to add a docstring for the new `return_unused_args` argument as well

Reply:
See https://github.com/huggingface/pytorch-transformers/pull/866#issuecomment-514245282